components/HeaderNav/HeaderNav.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
+// import * as React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import CreateEventForm from '../CreateEventForm/CreateEventForm';
 import { StyleSheet, View, Text, Button } from 'react-native';
 
-export default function HeaderNav() {
+export default function HeaderNav({ navigation }) {
```

Comment:
curious as to what this does deconstructing the navigation as a prop..... i'll have to dig into the code base to really understand what's going on here